z3c/dependencychecker/modules.py

```diff
@@ -29,6 +29,7 @@
 
 FOLDERS_TO_IGNORE = ("node_modules", "__pycache__")
 
+HAS_REPORTED_CONDITIONAL_IMPORTS = False
```

Comment:
Why the "reported"? It might have to do with code you'll add later. For now, it sounds like `CONDITIONAL_IMPORTS_FOUND` could be clearer?